src/System.Net.NameResolution/src/project.lock.json

```diff
@@ -1,5 +1,5 @@
 {
-  "locked": true,
+  "locked": false,
```

Comment:
Switching all publicly released project.json.lock back to "locked" for completeness.
This isn't really important. The workflow we choose is to handpick files from sn2 and move them to a new branch off master for the official PR. These files won't be moved unless the build in the _official_ PR needs them.